runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/RPASections/RPAContent/RPAContent.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+import Typography from '#src-landing/components/Typography';
+
+import styles from './RPAContent.module.scss';
+
+const RPAContent: FC = () => {
+    const { translate } = useTranslations();
+
+    return (
+        <div className={styles.root}>
+            <div className={styles.contentWrapper}>
+                <Typography className={styles.title}>
+                    {translate('Landing.RPA.Title')}
+                </Typography>
+                <div className={styles.content}>
+                    <div className={styles.paragraph1}>
+                        <Typography>
```

Comment:
same thing here, use body variant, example use cases in other sections

Reply:
done